std/algorithm/setops.d

```diff
@@ -3,6 +3,20 @@
 This is a submodule of $(MREF std, algorithm).
 It contains generic algorithms that implement set operations.
 
+The functions $(LREF multiwayMerge), $(LREF multiwayUnion), $(LREF setDifference),
+$(LREF setIntersection), $(LREF setSymmetricDifference) expect a range of sorted
+ranges as input.
+
+All the algorithms, except (LREF multiwayMerge) and (LREF mutiwayUnion),
+are generalized to accept as input not only sets but also multisets:
+$(UL
+    $(UI when all ranges passed as input are sets, the result is going to be
```

Comment:
```
/dev/shm/dtest/work/repo/dlang.org/web/phobos-prerelease/std_algorithm_setops.html:157:<ul>    <!--UNDEFINED MACRO: "UI"-->
/dev/shm/dtest/work/repo/dlang.org/web/phobos-prerelease/std_algorithm_setops.html:158:    <!--UNDEFINED MACRO: "UI"-->
```

@RazvanN7 as we currently can't rebase PRs without breaking GH hooks, could you please fix this on your branch (should be `LI`) and resubmit a PR? Thanks!